.github/actions/templating/action.yml

```diff
@@ -0,0 +1,9 @@
+name: templating
+runs:
+  using: composite
+  steps:
+    - shell: bash
+      run: |
+        set -x
+        git checkout -B master && scripts/dev/makedist >/dev/null
```

Comment:
it seems that even with `makedist` the stubs are not regenerated, is that wanted behaviour?